app/org/sagebionetworks/bridge/models/schedules/TaskScheduler.java

```diff
@@ -0,0 +1,26 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+
+public interface TaskScheduler {
```

Comment:
Is there value in having this be a separate class from BaseTaskScheduler? Why not consolidate the two into a single abstract class?